libretroshare/src/services/p3msgservice.cc

```diff
@@ -2007,29 +2153,31 @@ bool p3MsgService::receiveGxsTransMail( const RsGxsId& authorId,
 		if( mRecentlyReceivedMessageHashes.find(hash) !=
 		        mRecentlyReceivedMessageHashes.end() )
 		{
-			std::cerr << __PRETTY_FUNCTION__ << " (II) receiving "
+			RsInfo() << __PRETTY_FUNCTION__ << " (II) receiving "
 			          << "message of hash " << hash << " more than once. "
 			          << "Probably it has arrived  before by other means."
 			          << std::endl;
 			return true;
 		}
-		mRecentlyReceivedMessageHashes[hash] = time(NULL);
+		mRecentlyReceivedMessageHashes[hash] =
+		        static_cast<uint32_t>(time(nullptr));
```

Comment:
seems to me we switched to rs_time() to avoid the need to cast.

Reply:
The problem is `std::map<Sha1CheckSum, uint32_t> mRecentlyReceivedMessageHashes;` are you suggesting we should change that uint32_t for an rstime_t ?
I would be happy to do that if you are sure that will not break retrocompatibility, I am quite sure that map is not sent over network, do you know if it is saved to disk?